nvtabular/ops/categorify.py

```diff
@@ -201,23 +215,58 @@ def __init__(
             raise ValueError(
                 "cannot use search_sorted=True with anything else than the default freq_threshold"
             )
+        if isinstance(num_buckets, dict):
+            columns = list(num_buckets)
+            self.num_buckets = num_buckets
+        elif isinstance(num_buckets, list):
+            assert columns is not None
+            assert len(columns) == len(num_buckets)
+            self.num_buckets = {col: nb for col, nb in zip(columns, num_buckets)}
+        elif isinstance(num_buckets, int) or num_buckets is None:
+            self.num_buckets = num_buckets
+            print(self.num_buckets)
+        elif num_buckets == 0 or 0 in num_buckets:
```

Comment:
@rjzamora good question. this is covering such case: `num_buckets={"author": 10, "engaging_user": 20, "platform":0}` or `num_buckets=0`, and raising the ValueError below otherwise:
```            raise ValueError(
                "For hashing num_buckets should be an int > 1, otherwise set it to None."
            )
```
Does it make sense? I am open to suggestions.

Reply:
I see - But you wouldn't make it into this code block for either of the cases you mentioned, because `num_buckets` would be an `int` or `dict`.  Perhaps you don't want this to be an `elif` (or want to check for zero values in the blocks above)?